### PR TITLE
Prs/add create cluster module

### LINKED ISSUE #31: [module] create cluster
We need a module, that creates the ceph cluster in k8s

### PR DESCRIPTION
**Pull request by brueggemann**
Closes #31

### CODE REVIEW

**Comment by brueggemann** [comment]
This PR is blocked by #36 because we need to create the ConfigMap before we can create the CephCluster resource

**Comment by brueggemann** [comment]
This is also blocked by #37

File: src/rookify/modules/create_configmap/main.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            raise ModuleException("Secret rook-ceph-mon already exists")
+
+        self.__create_configmap_definition()
+
+    def execute(self) -> None:
+        configmap = kubernetes.client.V1ConfigMap(
+            **self.machine.get_preflight_state("CreateConfigMapHandler").configmap
+        )
+
+        configmap = self.k8s.core_v1_api.create_namespaced_config_map(
```
Comment (boekhorstb1) [inline src/rookify/modules/create_configmap/main.py:74]: it could be better/more readable to use different variables instead of reassigning them

**Comment by boekhorstb1** [comment]
In general there is almost no logging in the module, should we add that maybe? @NotTheEvilOne

File: src/rookify/modules/create_configmap/main.py (hunk 1 above)
Comment (NotTheEvilOne) [reply src/rookify/modules/create_configmap/main.py]: Sorry, missed that comment (as it was for PR #43)

**Review by boekhorstb1 — APPROVED** [review]

**Pull request merged**